content/docs/context.md

```diff
@@ -196,6 +197,20 @@ Requires a [function as a child](/docs/render-props.html#using-props-other-than-
 > 
 > For more information about the 'function as a child' pattern, see [render props](/docs/render-props.html).
 
+### `Context.displayName` {#contextdisplayName}
```

Comment:
This doesn't get parsed properly it seems. Maybe only lowercase is allowed?

Reply:
Yeah it seems that all links are in lowercase, I changed it to lowercase and pushed it.